packages/remediations/src/steps/reviewSystems.js

```diff
@@ -0,0 +1,166 @@
+    ] : allNewSystems;
+
+    const onRefresh = (options, callback) => {
+        if (!callback && inventory && inventory.current) {
+            inventory.current.onRefreshData(options);
+        } else if (callback) {
+            callback(options);
+        }
+    };
+
+    const onSelect = (selected) => {
+        input.onChange(Array.from(selected.keys()));
+    };
+
+    const onSelectRows = (value) => {
+        dispatch({
+            type: TOGGLE_BULK_SELECT,
+            payload: value
+        });
+    };
+
+    return (
+        <Stack hasGutter>
+            <StackItem>
+                <TextContent>
+                    <Text>
+                        Review and optionally exclude systems from your selection.
+                    </Text>
+                </TextContent>
+            </StackItem>
+            <StackItem>
+                {
+                    <Router>
+                        <InventoryTable
+                            hideFilters={{ all: true }}
+                            noDetail
+                            variant="compact"
+                            showTags
+                            onRefresh={onRefresh}
+                            ref={inventory}
+                            getEntities={async (_i, config) => {
+                                const invData = await inventoryApi?.getEntities(allSystems, {
+                                    ...config,
+                                    hasItems: true
+                                });
+                                const data = await inventoryApi?.mapTags(invData);
+                                return data;
+                            }}
+                            onLoad={({ mergeWithEntities, api }) => {
+                                registry.register(mergeWithEntities(entitiesReducer(onSelect)));
+                                setInventoryApi(() => api);
+                            }
+                            }
+                            bulkSelect={{
+                                id: 'select-systems',
+                                count: selected.size,
+                                items: [{
+                                    title: 'Select none (0)',
+                                    onClick: () => onSelectRows(false)
+                                },
+                                {
+                                    ...loaded && rows && rows.length > 0 ? {
+                                        title: `Select page (${ rows.length })`,
+                                        onClick: () => {
+                                            onSelectRows(true);
+                                        }
+                                    } : {}
+                                }],
+                                checked: selected.size > 0,
+                                onSelect: (value) => {
+                                    onSelectRows(value);
+                                }
```

Comment:
Redundant function wrapper
```suggestion
                                onSelect: (value) => onSelectRows(value)
```